### ISSUE TITLE
Preserve geometry when transforming CZML entities

### ISSUE THREAD

**Issue by lilleyse** [AnalyticalGraphicsInc/cesium#7397]
CZML entities like boxes can be optimized so that they don't recreate their geometry during translation/rotation/scale animations. This would help speed up czmls containing a large number these entities.

[Example CZML](https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/#c=tVVdT9swFP0rUV/opOLaiVM7UNBQQRpaWRntJg2CkJsY8HDiynHKx8R/n5OmJWm7TTwsD3bic++51+f6OnOmneglkc6BcxWmjvOrGBwnbIk4bO3ZOVZRnvDUhK3OEppznQmVLnAEYA2KpIoeCqDiKZlSw/WcyYW9CxHdhWQXBRPk7yG6B+Fld2PRg5cr1pI319omMREJ/zNNwyPJpREzKbguHPwaoll6V7EMR6Pzm/FkdN5wzQyfLfDxj/Hk5OxmMBwNPt+cfRtOTs+HpycXYWth/FpMr52tuk00ix54PJr+5FFdvKl6CltOTaBiMbb7SgtNs3Ws2j7ThmeClZpfYVg8AHYcb/XmV2/Xb66vnXqIWyFlSX7LZMYbkMqNFCkvUaPzreBASaW3J6fvpqxErlzf7zjlAMu5nkyl2EqHmcqEqU7R+mGZKckK7EjeKS3MfVJV6/TLydFFo1S3Sj8yHZ88Gc2WXpPnWVXeT6PhcfMYNXSsb/Mf58lxXBf1PEyBHV2vhyHxauAuJl6JBj4lxMcE1UAMKXIpBr7v0oC6MPA7f43srUVGReReAJDrBhhjFJCNyIQA3CMIEoTpemQMAaUUU59AsoSu16thZbbtxbYUJE+F+ZozW5W0At+nGwQepBgi5Hkk8GgQBE3QRZYBQdclELm9Xn1rEGCf+hC7KKA2eRy4uOFLerTn2b35hCLP2rxP1v+WGNpIbF328toI0+v9MA3Tub2B54I/cm3v4JQ/OgN7QvMEfC/X2jtR+TlQtjy2D/VOZ1md7F7lMj5KRWKrU7VumL5+2F9wxsywscp1xM+1SkTGLX1FPbAX/vEKBlKxuF38BErXMix4884Ai+P2BlvN9kWpZKK2mbQ6rX5mniU/XKT8USQzpY2Ta9kGoGt4MrM9y7PuNLeXpQFRlhW8/e7SqR+LuSPiA9u6TRnshRNJlmUWuc2lHIsX2/OH/a61b7gVexPp3cj+syR7Lkzu0eFwsQgA6Hft56aXUUpOma4x/gY) - you can check in the debugger that a new `BoxOutlineGeometry` is created per frame.

**Comment by hpinkos** [comment]
Ellipsoid already does something like this.  We should be able to apply the same logic to boxes and cylinders.

**Comment by mramato** [comment]
Yep, I mentioned this to @lilleyse offline.  It won't even be that hard to do, you can just use the DynamicGeometryUpdater in EllipsoidGeometryUpdater as a template.
